### PR TITLE
fix #83132 discudemy.com

### LINKED ISSUE #83132: discudemy.com
***Description***: https://www.discudemy.com/language/english has ad leftover.

<details><summary>Screenshots:</summary>

image
image

</details><br/>

<details><summary>Filter:</summary>

image

</details><br/>

### PR DESCRIPTION
**Pull request by wout-codes**
#83132

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/83132
+discudemy.com#?#section.card:has(> label:contains(/^Ads$/) ~ ins.adsbygoogle)
```
Comment (zloyden) [inline EnglishFilter/sections/css_extended.txt:5]: I'd prefer to use `:upward` -> `discudemy.com#?#article > section.card > ins.adsbygoogle:upward(section.card)`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: `:has` is faster than `:upward`.

Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt:5]: > but upward may better due less checking section elements

@krystian3w [said](https://github.com/AdguardTeam/AdguardFilters/issues/83132#issuecomment-840610710) a good thing.
@wout-codes about your rule. You're using double ext-css with Regex that should be avoided if possible.

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: I think it's more safe, but I will add yours.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt]: > I think it's more safe

This is true, but it is not necessary here.

**Pull request merged**